mitmproxy/io/compat.py

```diff
@@ -300,6 +300,11 @@ def convert_11_12(data):
     return data
 
 
+def convert_12_13(data):
+    data["version"] = 13
+    return data
```

Comment:
Here we then need to do a 
```python
if flow["marked"]:
    flow["marked"] = SYMBOL_MARK
else:
    flow["marked"] = ""
```
conversion to convert from bool to str when importing old dumpfiles.

Reply:
Done.